robot_skills/src/robot_skills/arm/guarded_motion.py

```diff
@@ -0,0 +1,40 @@
+import functools
+import rospy
+import types
+
+from control_msgs.msg import FollowJointTrajectoryGoal, FollowJointTrajectoryAction
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+
+def AddForceSensingArm(arm):
+    arm.move_down_until_force_sensor_edge_up = types.MethodType(move_down_until_force_sensor_edge_up, arm)
```

Comment:
Shouldn't this then be arm.force_sensor.move_down_until_force_sensor_edge_up?

Reply:
No, since it is the arm that is doing the moving down.